backends/gstreamer/src/player.rs

```diff
@@ -145,6 +149,18 @@ impl PlayerInner {
         Ok(())
     }
 
+    pub fn set_rate(&mut self, rate: f64) -> Result<(), BackendError> {
+        self.rate = rate;
```

Comment:
One final change, let's add this comment here:

_This method may be called before the player setup is done, so we safe the rate value and set it once the player is ready and after getting the media info_